src/containers/Home/homeReducer.js

```diff
@@ -0,0 +1,151 @@
+const initialState = {
+    reminders:[], 
+    error:null, 
+}
+
+
+export default function homeReducer (state = initialState , action ){
+   switch(action.type){
+        case "FETCH_REMINDER_PENDING": {
+            return {
+                ...state,
+                fetching:true
+            }
+        }
+        case "FETCH_REMINDER_REJECTED": {
+            return {
+                ...state,
+                fetching:false,
```

Comment:
I don't know what is the use of this `fetching` here, I think this for showing `loader` or something while making API call but I don't see any loader or anything like that while making API call, so what is the use of this?

If you are not using it, please remove it.